app/containers/Forms/Form.tsx

```diff
@@ -98,6 +98,31 @@ export const FormComponent: React.FunctionComponent<Props> = ({
       return result?.requiresEmissionAllocation === true;
     });
 
+  const customValidation = (formData, errors) => {
+    let hasFalseRequiresEmissionAllocation = false;
+    let nonEnergyProductCount = 0;
+    const productsInConflict = [];
+    if (formData[0]?.productRowId) {
+      formData.forEach((product, index: number) => {
+        if (product.requiresEmissionAllocation === false)
+          hasFalseRequiresEmissionAllocation = true;
+        if (product.productRowId > 7) {
+          nonEnergyProductCount++;
+          productsInConflict.push(index + 1);
+        }
+      });
+    }
+
+    if (hasFalseRequiresEmissionAllocation && nonEnergyProductCount > 1)
+      errors['0'].addError(
+        `More than one non-energy product that does not require allocation of emissions cannot be reported together. -- Products in conflict: ${productsInConflict.join(
+          ','
+        )}`
```

Comment:
```suggestion
        `${productsInConflict.join(
          ','
        )} cannot be reported together as at least one of these products does not require manual allocation of emissions`
```

I feel like this would make more sense